DependencyInjection/FOSUserExtension.php

```diff
@@ -57,7 +57,8 @@ public function load(array $configs, ContainerBuilder $container)
 
         // Set the SecurityContext for Symfony <2.6
         // Should go back to simple xml configuration after <2.6 support
-        if (interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
+        if (interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')
+            && $container->hasDefinition('security.token_storage')) {
```

Comment:
@xabbuh, right, true; well, then I'll be closing this and creating a new PR with a compiler pass then. Thanks for noting this!